### PR TITLE
Update sura names

### LINKED ISSUE #1739: Change Surahs name in Surah list and in page header
السلام عليكم ورحمة الله وبركاته 
I hope we change the Surah names format in Surahs list and page header specially Arabic language (with Tashkil as real Mushaf)to new format as the table attached : [Surahs_names](https://drive.google.com/file/d/12hMJfqhlbd6o9Ekmi8co9-lPwwmBNlOb/view?usp=drivesdk)  

سُورَةُ الفَاتِحَةِ Al-Fâtihah AL-FÃTIHAH
سُورَةُ آلِ عِمۡرَانَ Âl-`Imrân ÃL-`IMRÃN

وجزاكم الله خيرا

### PR DESCRIPTION
**Pull request by ahmedre**
Fixes #1739.

### CODE REVIEW

File: app/src/main/res/values-fr/sura_names.xml [hunk 1]
```diff
@@ -1,5 +1,122 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
+  <string-array name="sura_names">
+    <item>AL-FÃTIHAH</item>
```
Comment (ahmedre) [inline app/src/main/res/values-fr/sura_names.xml:4]: @nacer80 are these intentionally caps for people in France?

File: app/src/main/res/values-fr/sura_names.xml [hunk 2]
```diff
@@ -1,5 +1,122 @@
...
+    <item>AL-ISRÃ\'</item>
+    <item>AL-KAHF</item>
+    <item>MARЇAM</item>
+    <item>TÃ-HÃ</item>
+    <item>AL-ANBIYÃ\'</item>
+    <item>AL-HAJJ</item>
+    <item>AL-MU\'MINOUN</item>
+    <item>AN-NOUR</item>
+    <item>AL-FURQÃN</item>
+    <item>ACH-CHU`ARÃ\'</item>
```
Comment (ahmedre) [inline app/src/main/res/values-fr/sura_names.xml:29]: @nacer80 also, do all people who speak French use the CH instead of sh?

**Comment by ahmedre** [comment]
not sure if arabic is looking too crowded now though
arabic

**Comment by ahmedre** [comment]
cc @benomaire for review

**Comment by nacer80** [comment]
السلام عليكم ورحمة الله 
How about "go to page" setting.
When we need to go to Surah, we have to write it with tashkil?

**Comment by ahmedre** [comment]
وعليكم السلام ورحمة الله وبركاته
good point - should check that to make sure

**Comment by nacer80** [comment]
Before we had an issue with "Hamza"
such as Surah "الإنسان" we have write with Hamza

File: app/src/main/res/values-fr/sura_names.xml (hunk 2 above)
Comment (nacer80) [reply app/src/main/res/values-fr/sura_names.xml:29]: Yes, in Franch language they use "ch" it pronanced as "sh" in English

File: app/src/main/res/values-fr/sura_names.xml (hunk 1 above)
Comment (nacer80) [reply app/src/main/res/values-fr/sura_names.xml:4]: We can use also lowercase, but the Mushaf printed by KFGQPC they used uppercase.


Also here they use uppercase 
https://quranenc.com/ar/browse/french_montada

**Comment by ahmedre** [comment]
this still has a hamza problem that I have to fix

**Comment by ahmedre** [comment]
fixed the hamza problem - but if you search with a hamza (i.e. if you type الإنسان), it won't find it. I guess I can / should fix that, but guessing this will be more rare

**Comment by ahmedre** [comment]
fixed that also - the downside now is searching for الإ will return sura names where the alif is not with a kasra because we're just ignoring them - but I think this is ok.

with_cleanup

File: app/src/main/res/values-fr/sura_names.xml (hunk 1 above)
Comment (benomaire) [reply app/src/main/res/values-fr/sura_names.xml:4]: Related: Surah names used by NOOR International: https://www.noorinternational.net/fr/translation/swarlist
i did not check if they match the proposed names here

Comment (ahmedre) [reply app/src/main/res/values-fr/sura_names.xml:4]: Noor International's is a bit different than the ones from @nacer80 - but I'll go ahead and merge. please let me know if there are any I should change afterwards.

**Pull request merged**